### PR TITLE
Use generate_cert.py to generate ssl certificates

### PR DESCRIPTION
**Pull request by dburr**
Didn't realize there was already a python script that generates the ssl certificates.

Also made a few other enhancements:
* VM has a fixed MAC address, so that it is less likely to get assigned a different IP address by DHCP routers (and/or it is now possible to create a DHCP reservation so that the VM always gets the same IP address)
* setupServer.sh script is less chatty
* if a SSL certificate was created previously and still exists on disk, use that one instead of creating a new one.

### CODE REVIEW

File: generate_cert.py [hunk 1]
```diff
@@ -64,6 +65,7 @@
     datetime.datetime.utcnow() + datetime.timedelta(days=2 * 365)
 ).add_extension(
     x509.SubjectAlternativeName([
+        x509.DNSName(u"android.magica-us.com"),
```
Comment (SuzukaGozen) [inline generate_cert.py]: Is this actually needed? We have wildcard `*.magica-us.com` below.

Comment (dburr) [reply generate_cert.py]: oops... accidentally left in some debug code. just pushed another commit to yeet it out of existence.

File: setupServer.sh [hunk 2]
```diff
@@ -41,30 +42,31 @@ sed -e 's/^daemon/#daemon/' -i.bak /etc/nginx/nginx.conf
...
   cp "$FILE" /etc/nginx/cert/
 done
 # start nginx
 systemctl start nginx.service
 
 # install dependencies etc
 echo ""
-echo "# Installing ZipZap python dependencies..."
-cd $SRC && pip3 install -r requirements.txt
+echo "# Installikng ZipZap python dependencies..."
```
Comment (SuzukaGozen) [inline setupServer.sh]: Typo `Installikng`

Comment (dburr) [reply setupServer.sh]: Doh!

**Pull request merged**